src/dataflow-types/src/client/controller.rs

```diff
@@ -34,6 +34,7 @@ use timely::progress::Timestamp;
 use tokio_stream::StreamMap;
 
 use mz_orchestrator::{CpuLimit, MemoryLimit, Orchestrator, ServiceConfig, ServicePort};
+use mz_persist_types::Codec64;
```

Comment:
This was added as a requirement to lots of generic arguments, but I couldn't see where it was required. Can you explain?

Reply:
Resolved by hiding the `ReadHandle`, which was causing this behind a trait object.

Reply:
Resolved by hiding the `ReadHandle`, which was causing this behind a trait object.